### PR TITLE
`chore`: Integration tests for Advanced backup/restore scenarios

### PR DESCRIPTION
**Pull request by SiddeshUbale**
**This PR contains Integration tests for:**

1] Deleted namespace test.
2] illegal namespace test (more than 127 namespaces).

**Known Improvements:-**

Function for executing all 'GQL Request With AccessJwt token'.

### CODE REVIEW

File: systest/backup/advanced-scenarios/127-Namespace/backup_test.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+	ShellToUse      = "bash"
+)
+
+func TestDeletedNamespaceID(t *testing.T) {
+	err := common.RemoveContentsOfPerticularDir(t, "./data/backup/")
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+	// *****************Get JWT-Token
```
Comment (matthewmcneely) [inline systest/backup/advanced-scenarios/127-Namespace/backup_test.go:32]: I find these asterisk-based lines distracting

File: systest/backup/advanced-scenarios/127-Namespace/backup_test.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+	jwtTokenAlpha1 := testutil.GrootHttpLogin("http://" + testutil.SockAddrHttp + "/admin").AccessJwt
+	headerAlpha1.Set(accessJwtHeader, jwtTokenAlpha1)
+	fmt.Println("Header1: ", jwtTokenAlpha1)
```
Comment (matthewmcneely) [inline systest/backup/advanced-scenarios/127-Namespace/backup_test.go:36]: Were these leftover from when you were writing the tests? In most cases, tests should be silent unless failed.

File: systest/backup/common/utils.go [hunk 3]
```diff
@@ -38,6 +49,12 @@ var (
 		"alpha2",
 		"alpha3",
 	}
+	namespaceId uint64
+)
+
+const (
+	accessJwtHeader = "X-Dgraph-AccessToken"
+	ShellToUse      = "bash"
```
Comment (matthewmcneely) [inline systest/backup/common/utils.go:57]: Does the scope of this constant need to be made public?

**Review by matthewmcneely — CHANGES_REQUESTED** [review]

File: systest/backup/advanced-scenarios/127-Namespace/backup_test.go (hunk 1 above)
Comment (SiddeshUbale) [reply systest/backup/advanced-scenarios/127-Namespace/backup_test.go]: **Resolved:**  Removed all distracting comments from the code.

File: systest/backup/advanced-scenarios/127-Namespace/backup_test.go (hunk 2 above)
Comment (SiddeshUbale) [reply systest/backup/advanced-scenarios/127-Namespace/backup_test.go]: **Resolved:** Removed excessive print statements from the code.

File: systest/backup/common/utils.go (hunk 3 above)
Comment (SiddeshUbale) [reply systest/backup/common/utils.go]: **Resolved:** There is no need to make these constants public.

**Review by matthewmcneely — CHANGES_REQUESTED** [review]

**Pull request closed without merging**